cmd/syft/cli/options/packages.go

```diff
@@ -0,0 +1,118 @@
+package options
+
+import (
+	"fmt"
+
+	"github.com/anchore/syft/internal/formats/table"
+	"github.com/anchore/syft/syft"
+	"github.com/anchore/syft/syft/pkg/cataloger"
+	"github.com/anchore/syft/syft/source"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+type PackagesOptions struct {
```

Comment:
refactor and remove these =)

Reply:
We still need the hard values here for decoding the flags as well as a receiver for the method. There is a future refactor that possibly removes these, but it's not for this PR.